### PR TITLE
Add absent cache metrics

### PR DESCRIPTION
**Pull request by yuzhu**
Add some metrics for the absent cache

### CODE REVIEW

**Review by calvinjia — CHANGES_REQUESTED** [review]
Should we also track number of invalidations?

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -5287,7 +5287,7 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.master.journal.init.from.backup";
     public static final String MASTER_JOURNAL_TOLERATE_CORRUPTION
         = "alluxio.master.journal.tolerate.corruption";
-    public static final String MASTER_JOURNAL_TYPE = "alluxio.master.journal.type";
+    public static final String MASTER_JOURNAL_TYPE = "alluxio.master.journal.typelog";
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:5290]: Is this change intended?

Comment (yuzhu) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java]: oops fixed.

**Review by calvinjia — APPROVED** [review]

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**